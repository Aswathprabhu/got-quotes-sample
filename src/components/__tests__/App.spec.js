import App from '../App.vue';
import { render } from '@testing-library/vue';

describe('App Page - Initial Render', () => {
  it('Initial render', () => {
    const { container } = render(App, {
      stubs: ['router-view'],
    });
    expect(container).toMatchSnapshot();
  });
});
