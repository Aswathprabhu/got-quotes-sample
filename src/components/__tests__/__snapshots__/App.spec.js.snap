// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Page - Initial Render Initial render 1`] = `
<div>
  <div
    id="app"
  >
    <router-view-stub />
  </div>
</div>
`;
